.github/actions/e2e_cleanup_timeframe/e2e-cleanup.sh

```diff
@@ -14,20 +14,20 @@ function download_tfstate_artifact {
 # delete_resources runs terraform destroy on the constellation-terraform subfolder of a given folder.
 function delete_resources {
   if [[ -d "$1/constellation-terraform" ]]; then
-    cd "$1/constellation-terraform" || exit 1
-    terraform init > /dev/null || exit 1 # first, install plugins
-    terraform destroy -auto-approve || exit 1
-    cd ../../ || exit 1
+    cd "$1/constellation-terraform" || return 1
+    terraform init > /dev/null || return 1 # first, install plugins
+    terraform destroy -auto-approve || return 1
+    cd ../../ || return 1
```

Comment:
we need to make sure that we leave the directory if one of the terraform commands fail